perfact/zodbsync/helpers.py

```diff
@@ -28,19 +18,17 @@ def to_string(value, enc='utf-8'):
     '''This method delivers bytes in python2 and unicode in python3.'''
```

Comment:
Well I guess the docstring does not really fit anymore

Reply:
I guess you are right - maybe we don't need these wrappers anymore...

Reply:
At least the `to_ustring` wrapper can go, it is not actually used anymore.

Reply:
We definitely don't. The wrappers were actually more to make work easier in Zope2's Restricted Python, later to make it easier to keep python-perfact Py2-Py3-compatible. Then they started creeping into all kinds of code, like `safe_syscall()` did…